### PR TITLE
Fix lots of spelling errors throughout yarn

### PR DESCRIPTION
**Pull request by Earthcomputer**
I ran yarn through a spellchecker.
Note that this doesn't fix anything in the realms package, as some of those spelling errors are due to odd Mojang conventions that should really be left to another PR to fix.

### CODE REVIEW

**Review by Ueaj-Kerman — APPROVED** [review]

**Review by Gegy — APPROVED** [review]

**Review by jaskarth — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Review by Prospector — APPROVED** [review]
Looks good! Glad to have all this stuff fixed. Also 69 files. Nice.

**Comment by Earthcomputer** [comment]
> Also 69 files. Nice.

Another reason not to do realmsclient yet :D

**Review by modmuss50 — APPROVED** [review]
Awesome work 👍

File: mappings/net/minecraft/world/storage/RegionFile.mapping [hunk 1]
```diff
@@ -11,12 +11,12 @@ CLASS net/minecraft/class_2861 net/minecraft/world/storage/RegionFile
 	METHOD <init> (Ljava/io/File;Ljava/io/File;Z)V
 		ARG 1 file
 		ARG 2 directory
-		ARG 3 dsync
+		ARG 3 desync
```
Comment (Runemoro) [inline mappings/net/minecraft/world/storage/RegionFile.mapping:14]: This is not `desync`. It actually *synchronizes* the data to the **d**isk as soon as it's written, by passing the `StandardOpenOption.DSYNC` option when opening the file. See this discussion when it was added:

https://github.com/FabricMC/yarn/pull/1253#discussion_r402636940

Comment (liach) [reply mappings/net/minecraft/world/storage/RegionFile.mapping:14]: Guess we will fix this in some other pr later

**Pull request merged**